src/App.js

```diff
@@ -1,26 +1,68 @@
-import activities from './resources/schedule.json'
+import activities from "./resources/schedule.json";
+import CurrentWeek from "./components/CurrentWeek";
+import ListedActivities from "./components/ListedActivities";
+import { useMemo } from "react";
 import dayjs from "dayjs";
 
 function App() {
+  const today = dayjs().format("MDYYYY");
+  var tomorrow = dayjs().add(1,'day');
+  var remDays = 6 - tomorrow.day();
+  var currentWeekStart = dayjs().add(2,'day');
+  var currentWeekEnd = dayjs().add(remDays+1,'day');
```

Comment:
dayjs has a utility to get the week's end. here: https://day.js.org/docs/en/manipulate/end-of